src/routes/safe/components/Transactions/TxsTable/index.jsx

```diff
@@ -32,6 +38,7 @@ const expandCellStyle = {
 type Props = {
   classes: Object,
   transactions: List<Transaction>,
+  incomingTransactions: List<IncomingTransaction>,
```

Comment:
I think you can move merging logic to the selector

Reply:
@mikheevm, so it would be to provide one selector with both tx types merged in a collection?

Then how would you resolve the table content build? (https://github.com/gnosis/safe-react/pull/333/files/bbaf2f6bbb82fd599e3f7cb3909b1962e19b274e#diff-77dae8f1c46a95ff0bbb364a4ca07a03R71)

Reply:
Sorry, I don't see the problem. Could you please give some more details?

Reply:
I think I got it. I think you can modify the function `getTableData` to do something like this:
```
if incoming tx:
  return this
else outgoing tx:
  return that
```

Reply:
@mikheevm, and... it's done. Please check/validate.